picocom.c

```diff
@@ -1813,6 +1821,25 @@ parse_args(int argc, char *argv[])
                 r = -1;
             }
             break;
+	case 'R':
+	    {
+                int rate = atoi(optarg);
+
+		if (rate < 0) {
+                    fprintf(stderr, "Invalid --rate: %d\n", rate);
+                    r = -1;
+		} else if (rate == 0) {
+		    tty_sleep.tv_sec = 0;
+		    tty_sleep.tv_nsec = 0;
+		} else if (rate == 1) {
+		    tty_sleep.tv_sec = 1;
+		    tty_sleep.tv_nsec = 0;
```

Comment:
... this checking becomes much simpler because -T0 simply disables it and everything else is the delay. Probably with some upprt limit.